src/logics/stargate/configs.ts

```diff
@@ -0,0 +1,592 @@
+          { chainId: common.ChainId.avalanche, poolIds: [PoolId.USDC, PoolId.USDT] },
+        ],
+      },
+      {
+        id: PoolId.USDT,
+        token: polygonTokens.USDT,
+        decimals: 6,
+        paths: [
+          { chainId: common.ChainId.mainnet, poolIds: [PoolId.USDC, PoolId.USDT] },
+          { chainId: common.ChainId.optimism, poolIds: [PoolId.USDC] },
+          { chainId: common.ChainId.base, poolIds: [PoolId.USDC] },
+          { chainId: common.ChainId.arbitrum, poolIds: [PoolId.USDC, PoolId.USDT] },
+          { chainId: common.ChainId.avalanche, poolIds: [PoolId.USDC, PoolId.USDT] },
+        ],
+      },
+      {
+        id: PoolId.DAI,
+        token: polygonTokens.DAI,
+        decimals: 6,
+        paths: [
+          { chainId: common.ChainId.mainnet, poolIds: [PoolId.DAI] },
+          { chainId: common.ChainId.optimism, poolIds: [PoolId.DAI] },
+        ],
+      },
+      {
+        id: PoolId.MAI,
+        token: polygonTokens.miMATIC,
+        decimals: 6,
+        paths: [
+          { chainId: common.ChainId.mainnet, poolIds: [PoolId.MAI] },
+          { chainId: common.ChainId.optimism, poolIds: [PoolId.MAI] },
+          { chainId: common.ChainId.arbitrum, poolIds: [PoolId.MAI] },
+          { chainId: common.ChainId.avalanche, poolIds: [PoolId.MAI] },
+        ],
+      },
+    ],
+  },
+  {
+    id: ChainId.METIS,
+    chainId: common.ChainId.metis,
+    contract: {
+      Router: '0x2F6F07CDcf3588944Bf4C42aC74ff24bF56e7590',
+      RouterETH: '',
+      FeeLibrary: '0x55bDb4164D28FBaF0898e0eF14a589ac09Ac9970',
+      LayerZeroEndpoint: '0x9740FF91F1985D8d2B71494aE1A2f723bb3Ed9E4',
+    },
+    pools: [
+      {
+        id: PoolId.METIS,
+        token: metisTokens['METIS(ERC20)'],
+        decimals: 18,
+        paths: [{ chainId: common.ChainId.mainnet, poolIds: [PoolId.METIS] }],
+      },
+      {
+        id: PoolId.metisUSDT,
+        token: metisTokens['m.USDT'],
+        decimals: 6,
+        paths: [
+          { chainId: common.ChainId.mainnet, poolIds: [PoolId.metisUSDT] },
+          { chainId: common.ChainId.avalanche, poolIds: [PoolId.metisUSDT] },
+        ],
+      },
+    ],
+  },
+  {
+    id: ChainId.BASE,
+    chainId: common.ChainId.base,
+    contract: {
+      Router: '0x45f1A95A4D3f3836523F5c83673c797f4d4d263B',
+      RouterETH: '0x50B6EbC2103BFEc165949CC946d739d5650d7ae4',
+      FeeLibrary: '0x9d1b1669c73b033dfe47ae5a0164ab96df25b944',
+      LayerZeroEndpoint: '0xb6319cC6c8c27A8F5dAF0dD3DF91EA35C4720dd7',
+    },
+    STG: baseTokens.STG,
+    pools: [
+      {
+        id: PoolId.USDC,
+        token: baseTokens.USDbC,
+        decimals: 6,
+        paths: [
+          { chainId: common.ChainId.mainnet, poolIds: [PoolId.USDC, PoolId.USDT] },
+          { chainId: common.ChainId.optimism, poolIds: [PoolId.USDC] },
+          { chainId: common.ChainId.polygon, poolIds: [PoolId.USDC] },
+          { chainId: common.ChainId.arbitrum, poolIds: [PoolId.USDC] },
```

Comment:
```suggestion
          { chainId: common.ChainId.polygon, poolIds: [PoolId.USDC, PoolId.USDT] },
          { chainId: common.ChainId.arbitrum, poolIds: [PoolId.USDC, PoolId.USDT] },
```

Reply:
from the Stargate UI, I didn't see the mapping. Can you illustrate where do you find it?

Reply:
Let's follow the UI as it seems their [docs](https://stargateprotocol.gitbook.io/stargate/developers/stargate-chain-paths) isn't up to date, thanks.